### DISCUSSION TITLE
New : error cannot read properties of undefined (reading ‘length’)

### DISCUSSION THREAD

**Discussion by dairen** [KevinVandy/material-react-table#1034] [Q&A]
Hi
I found your package and I love design and code

I follow basic example in JavaScript 
Const data
Const columns with useMemo
Const table

Before to add render, I have the error read properties 

I have all prerequisite 
React 18
React dom 18
...

I'm sorry to ask your help about this basic issue :)

**Comment by KevinVandy** [comment]
welp, you're not really sharing any of your code. How do you expect anyone to help you?
